### ISSUE TITLE
BUG: centralMass in dump

### ISSUE THREAD

**Issue by Anto6453** [idefix-code/idefix#286]
### Describe the issue:

When restarting from a different grid, I realized that the values given by `image->centralMass` (when restarting from a `DumpImage`) were totally random and didn't really make sense. I therefore have the impression that the `gravity.centralMass` variable is not correctly saved in the dump files.
I've added the line
```c++
data->dump->RegisterVariable(&this->centralMass, "centralMass",1);
``` 
just after line 31 in  [`gravity.cpp`](https://github.com/idefix-code/idefix/blob/a92732f3683af37bc3e55a9e8954867b1ae43967/src/gravity/gravity.cpp) and it seems to solve the problem.

### runtime information:

Idefix version : [V2.1.01](https://github.com/idefix-code/idefix/commit/767fbcd7946510c1341d65dccb168745b7d2270d)

**Comment by glesur** [comment]
That's correct, thanks @Anto6453 . Can you open a PR with this fix, or do you want me to do it?

**Comment by glesur** [comment]
fixed in #287
